tox.ini

```diff
@@ -12,24 +12,32 @@ deps =
     edx-opaque-keys
     mock
     django_nose>=1.4
+    xblockutils
```

Comment:
Ah, we're installing directly from `setup.py`; cool.
Can we add most of these deps there?
Then we can just add the "dynamic" ones here:

```ini
deps =
    -rrequirements.txt
    django >= 1.8, < 1.9
```